lib/utils/messages.js

```diff
@@ -4,17 +4,25 @@ var _ = require('lodash');
 var colors = require('colors/safe');
 
 var TOOL_NAME = 'api-sync';
+var requiresSetup = '\n\nRequires setup: Do it with ' + TOOL_NAME +
```

Comment:
I think requiresSetup should only contain the message. The line end characters should be optionally included in the places where requiresSetup is used.